keyboards/wilba_tech/rama_works_m60_a/keymaps/zyber/config.h

```diff
@@ -0,0 +1,5 @@
+// USB Device descriptor parameter
+#undef PRODUCT
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

And an `#pragma once`